features/localization/managing_countries/editing_country.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country
+    In order to enable or disable countries
+    As an Administrator
+    I want to be able to edit country
+
+    Background:
+        And I am logged in as administrator
+
+    @todo
+    Scenario: Disabling country
+        Given the store has "France" country enabled
+        And I want to disable country
```

Comment:
I think toggling is so specific function, that it is worth to mention the beginning state of entity. I vote for 2 different steps.